libs/lambda-edge-openid-auth/src/lib/views/bad-request.ts

```diff
@@ -22,32 +22,16 @@ export function badRequest(config: Config): CloudFrontResultResponse {
         statusDescription: 'Bad Request',
         body: page,
         headers: {
-            'set-cookie': [
-                {
+            'set-cookie': ['TOKEN', 'REFRESH_TOKEN', 'NONCE', 'IDP'].map(
```

Comment:
I know it's the previous behaviour but is it required to clear cookies on a bad request?

Reply:
oic - it's really a malformed / empty token that generates a BadRequest - maybe it should be renamed?

Reply:
Not required, but we added it in because if they get into a bad state. They wouldn't have to clear the cookies, it would be reset, and a new login flow will be initiated.